public/js/contribute/pad/load.js

```diff
@@ -69,22 +68,24 @@ async function DOMLoad() {
       });
 
       const url = new URL(window.location);
-      if (!queryparams) var queryparams = new URLSearchParams(url.search);
-      queryparams.delete('display');
+      if (!window.queryparams)
+        window.queryparams = new URLSearchParams(url.search);
+      window.queryparams.delete('display');
 
       d3.select('div.display-source a').attr(
         'href',
-        `?${queryparams.toString()}`,
+        `?${window.queryparams.toString()}`,
       );
     }
   } else if (d3.select('div.display-option.display-source').node()) {
     const url = new URL(window.location);
-    if (!queryparams) var queryparams = new URLSearchParams(url.search);
-    queryparams.set('display', 'adjacent-source');
+    if (!window.queryparams)
+      window.queryparams = new URLSearchParams(url.search);
+    window.queryparams.set('display', 'adjacent-source');
```

Comment:
and here we are adding display contradicting the code above. should really be two separate variables